app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -185,7 +185,16 @@
                   canRemoveItem=false
                 }}
               {{else}}
-                {{input type=field.type value=(mut (get data.speaker field.fieldIdentifier)) id=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))}}
+                {{#if (eq field.name 'Twitter')}}
```

Comment:
@kushthedude I think you can use condition `if (or (eq field.name 'Website') (eq field.name 'Twitter'))` on line 180 than duplication of code here.